### PR TITLE
Made ReadMe file for KUOSC.

### PR DESCRIPTION
**Pull request by felixop7**
Made a readme file for KUOSC.
#7 solved.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,70 @@
...
+- [Usage](#Usage)
+- [Contributing](#Contributing)
+- [Community](#Community)
+- [License](#License)
 
-```
-$ yarn
-```
+#Introduction
+The Kathmandu University Open Source Project (KUOSC) website aims to provide an open and collaborative platform for showcasing the innovative projects, research, and contributions made by the Kathmandu University community. By developing a user-friendly and informative website, we strive to create a space where students, faculty, and alumni can share their work, collaborate on projects, and foster a culture of open-source collaboration.
```
Comment (aadarshadhakalg) [inline README.md:18]: ** contributions made by community members **

File: README.md [hunk 2]
```diff
@@ -1,42 +1,70 @@
...
+Contributor Profiles: Learn about the individuals behind the projects and their contributions.
+Research Highlights: Discover groundbreaking research initiatives undertaken by the university's scholars.
+Event Announcements: Stay updated with upcoming seminars, workshops, and events organized by KUOSC.
+Easy Navigation: Intuitive user interface for seamless browsing and interaction.
+Open Source: The website itself is an open-source project, encouraging collaboration and improvements from the community.
 
-```
-$ yarn start
-```
+#Getting-Started
```
Comment (aadarshadhakalg) [inline README.md]: may be we can rename this to 'Before you build this site'?

File: README.md [hunk 3]
```diff
@@ -1,42 +1,70 @@
...
+Discord
+Twitter
+Mailing List
+We value your feedback and ideas to make this project thrive.
 
-If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
+#License
+The KUOSC website is open-source and available under the MIT License. You're free to use, modify, and distribute the project's code following the terms of the license.
+
+Thank you for your interest in the Kathmandu University Open Source Project (KUOSC) website. We hope this README provides a clear understanding of our project's purpose, features, installation, and contribution guidelines. Feel free to explore the repository, share your feedback, and join our community to contribute to open source at Kathmandu University!
```
Comment (aadarshadhakalg) [inline README.md:82]: We don't need this.

**Review by aadarshadhakalg — CHANGES_REQUESTED** [review]
Thank you for the PR @felixop7.  Please address the changes that have been requested.

File: README.md [hunk 4]
```diff
@@ -1,42 +1,70 @@
...
-$ yarn
-```
+#Introduction
+The Kathmandu University Open Source Project (KUOSC) website aims to provide an open and collaborative platform for showcasing the innovative projects, research, and contributions made by the Kathmandu University community. By developing a user-friendly and informative website, we strive to create a space where students, faculty, and alumni can share their work, collaborate on projects, and foster a culture of open-source collaboration.
 
-### Local Development
+#Features
+Project Showcase: Explore a diverse range of projects developed by the Kathmandu University community.
+Contributor Profiles: Learn about the individuals behind the projects and their contributions.
+Research Highlights: Discover groundbreaking research initiatives undertaken by the university's scholars.
```
[the remark this answers is not in the document]
Comment (aadarshadhakalg) [reply README.md]: Just the line no 22

**Comment by felixop7** [comment]
Done @aadarshadhakalg

**Comment by aadarshadhakalg** [comment]
@felixop7  Still some issues need to be addressed. If you can manage some more time to work on this that would be appreciated.

**Pull request closed without merging**